File: assignments_nilesh_sir/java_assignments/Assignment_1/MathUtil.java
public class MathUtil {

    int count = 0;

    public int countPrime(int n1, int n2) {
        for (int i = n1; i < n2; i++) {
            if (isPrime(i)) {
                count++;
            }
        }
        return count;
    }

    public boolean isPrime(int num) {
        if (num == 0 || num == 1) {
            return false;
        }
        if (num == 2 || num == 3 || num == 5) {
            return true;
        }
        if (num % 2 == 0 || num % 3 == 0 || num % 5 == 0) {
            return false;
        }
        for (int i = 5; i < num / 2; i += 6) {
            if (num % i == 0) {
                return false;
            }
        }
        return true;
    }

    public int add(int n1, int n2) {
        return n1 + n2;
    }

    public int subtract(int n1, int n2) {
        return n1 - n2;
    }

    public int multiply(int n1, int n2) {
        return n1 * n2;
    }
}

File: assignments_nilesh_sir/Dotnet_assignments/Assignment_1/MathUtil.cs
using System.Security.Cryptography.X509Certificates;

class MathUtil
{
    private static bool IsPrime(int num)
    {
        if (num == 1 || num == 0)
        {
            return false;
        }
        if (num == 2 || num == 3 || num == 5)
        {
            return true;
        }
        if (num % 2 == 0 || num % 3 == 0 || num % 5 == 0)
        {
            return false;
        }
        for (int i = 5; i < num / 2; i += 6)
        {
            if (num % i == 0)
            {
                return false;
            }
        }
        return true;
    }

    public static int CountPrime(int num1, int num2)
    {
        int Count = 0;
        for (int i = num1; i <= num2; i++)
        {
            if (IsPrime(i))
            {
                Count++;
            }
        }
        return Count;
    }

    public static int Add(int num1, int num2)
    {
        return num1 + num2;
    }

    public static int Subtract(int num1, int num2)
    {
        return num2 - num1;
    }

    public static int Multiply(int num1, int num2)
    {
        return num1 * num2;
    }
}